src/fiatconnect/saga.ts

```diff
@@ -40,21 +49,21 @@ export function* handleFetchFiatConnectQuotes({
 }: ReturnType<typeof fetchFiatConnectQuotes>) {
   const { flow, digitalAsset, cryptoAmount } = params
   const userLocation: UserLocationData = yield select(userLocationDataSelector)
-  const account: string = yield select(currentAccountSelector)!
   const localCurrency: LocalCurrencyCode = yield select(getLocalCurrencyCode)
   const fiatConnectCashInEnabled: boolean = yield select(fiatConnectCashInEnabledSelector)
   const fiatConnectCashOutEnabled: boolean = yield select(fiatConnectCashOutEnabledSelector)
+  const fiatConnectProviders: FiatConnectProviderInfo[] = yield select(fiatConnectProvidersSelector)
```

Comment:
Do we have to worry about a race condition if fetchProviders call is slow and this attempts to fetch quotes while the API call is still happening? Or is that an edge case that easily be handled by user retry?

Reply:
What would the behavior there be? Would an error actually be thrown? It looks like we'd just set the providers query param to empty, so not sure if this would generate an error/prompt for a retry.

Reply:
No it wouldn't. I'll add in functionality so that if the cache has never been populated that an error pops up and the user can refresh by going back and forward again.